README.md

```diff
@@ -62,14 +66,36 @@ There are _some_ sense to the default keybindings:
 Although initially designed for giving lectures, the whiteboard controls are pretty good.
 I might add support for extending infinitely in several directions, in the style of a regular whiteboard app.
 
+## Frequently asked questions
+
+> My saved file won't open. Help!
+
+There was a short period of time when we used a different file format for our JSON files.
+It's very unlikely that you have such a file.
+In case you do, you can make it compatible with the modern qboard app by taking the file and wrapping the contents like so:
+
+```json
+{
+  "qboard-version": 1,
+  "pages": OLD_FILE_CONTENTS_GO_HERE
+}
+```
```

Comment:
Do we want to use tabs for these indents? Tabs vs spaces in code is one thing but for such a document, for accessibility purposes, tabs may be better. Viewers can change tab width to their convenience.

Reply:
no strong opinion

Reply:
leaving as is so we don't need to fight with the reformatter. i doubt anyone will have any real trouble with spaces as is (they don't actually have to *read* the object).